public/shared.mjs

```diff
@@ -32,7 +32,7 @@ export const FEEDS = [
   'https://developers.google.com/web/updates/2017/rss.xml',
   'https://developers.google.com/web/fundamentals/rss.xml',
   'https://developers.google.com/web/tools/rss.xml', // Note: only last 20 items.
-  'https://developers.google.com/web/showcase/rss.xml', // Note: only last 20 items.
+  'https://developers.google.com/web/tools/rss.xml', // Note: only last 20 items.
```

Comment:
just updated this. can you keep showcase. We had 2 tools urls.

Reply:
Urm, sure. will re-update.

Reply:
done.